### PR TITLE
Fix #279: Provide with a config file to customise stop-words

### LINKED ISSUE #279: Search result not so great on DocFX
Before Kusto onboard to DocFX,  its search results were pretty good; however, the search experience is not so good on DocFX.
https://kusto.azurewebsites.net/docs/index.html

For example, when I tried to search for “let”, it shows no result…….
I have to search for “let statement” to find what I am looking for…..

### PR DESCRIPTION
**Pull request by qinezh**
@vicancy @superyyrrzz @vwxyzh  @chenkennt

### CODE REVIEW

**Comment by vwxyzh** [comment]
👍

**Comment by superyyrrzz** [comment]
LGTM

File: src/docfx.website.themes/default/styles/search-worker.js [hunk 1]
```diff
@@ -2,11 +2,25 @@
...
       this.ref('href');
       this.field('title', {boost: 50});
       this.field('keywords', {boost: 20});
   });
   lunr.tokenizer.seperator = /[\s\-\.]+/;
+
+  var stopWords = [];
+  var stopWordsRequest = new XMLHttpRequest();
+
+  stopWordsRequest.open('GET', '../stopwords.json');
```
Comment (vicancy) [inline src/docfx.website.themes/default/styles/search-worker.js]: add a `search-` prefix?

File: src/docfx.website.themes/default/styles/search-worker.js [hunk 2]
```diff
@@ -2,11 +2,25 @@
...
 
   var lunrIndex = lunr(function() {
+      this.pipeline.remove(lunr.stopWordFilter);
       this.ref('href');
       this.field('title', {boost: 50});
       this.field('keywords', {boost: 20});
   });
   lunr.tokenizer.seperator = /[\s\-\.]+/;
+
+  var stopWords = [];
```
Comment (vicancy) [inline src/docfx.website.themes/default/styles/search-worker.js]: no need?

**Comment by vicancy** [comment]
:+1:

**Pull request merged**